Runtime/Scripts/BaseMeshSync.Textures.cs

```diff
@@ -0,0 +1,226 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using UnityEditor;
```

Comment:
This is in the runtime namespace, we might need to use preprocessor directives for callbacks from UnityEditor, otherwise builds will fail

Reply:
Fixed